<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Sumathi" failures="1" tests="2" name="testRunner.TestNGTestRunner" time="17.101" errors="0" timestamp="2022-05-25T15:38:43 MDT" skipped="0">
  <testcase classname="testRunner.TestNGTestRunner" name="runScenario" time="10.353">
    <failure message="expected [null] but found [Tomato]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [null] but found [Tomato]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at stepDefinitions.offerStepDef.user_check_if_the_product_exists(offerStepDef.java:55)
at âœ½.User check if the product exists(file:///C:/Users/nsuma/OneDrive/Desktop/Selenium%20Workspace/Cucumber/src/test/java/Feature/Offer.feature:12)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="testRunner.TestNGTestRunner" name="runScenario" time="6.748"/>
  <system-out/>
</testsuite> <!-- testRunner.TestNGTestRunner -->
